Public/Src/Sandbox/MacOs/Sandbox/Src/Kauth/TrustedBsdHandler.cpp

```diff
@@ -19,24 +19,24 @@ int TrustedBsdHandler::HandleLookup(const char *path)
     return KERN_SUCCESS;
 }
 
-int TrustedBsdHandler::HandleReadlink(vnode_t symlinkVNode)
+int TrustedBsdHandler::HandleReadVnode(vnode_t vnode, FileOperation operationToReport, bool isVnodeDir)
 {
     // get symlink path
     char path[MAXPATHLEN];
     int len = MAXPATHLEN;
-    int err = vn_getpath(symlinkVNode, path, &len);
+    int err = vn_getpath(vnode, path, &len);
     if (err)
     {
-        log_error("Could not get VNnode path for readlink operation; error code: %#X", err);
+        log_error("Could not get VNnode path for %d operation; error code: %#X", operationToReport, err);
         return KERN_SUCCESS; // don't deny access because of our own error
     }
 
     // check read access
```

Comment:
Nit: remove comment

Reply:
which comment? `// don't deny access because of our own error`? I like that comment :)

Reply:
```// check read access``` I thought, as it does different types of read checking, but then again thats kind of vaguely accurate too. Feel free to ignore. ;)

Reply:
I think it's fairly accurate: it does use `Checkers::CheckRead` to check it, it's just that it is now parameterized by the original operation that caused the read access